release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1372,12 +1372,28 @@ In {product-title} {product-version}, {olmv1-first} changes its default upgrade
 For more information, see xref:../release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-olmv1-legacy-olm-upgrade-edges_release-notes[Support for {olmv0} upgrade edges in {olmv1} (Technology Preview)].
 
 [discrete]
-[id="ocp-4-16-builder-sa"]
-=== The builder service account is no longer created if the Build cluster capability is disabled
+[id="ocp-4-16-dal12-region_{context}"]
+=== Dal12 region added
 
-With this release, if you disable the `Build` cluster capability, the `builder` service account and its corresponding secrets are no longer created.
+With this release, the `dal12` region has been added to the Power VS Installer.
```

Comment:
I would use "was added"

Reply:
Also please double-check if "Power VS installer" is the right term here, as opposed to any of the existing attributes around "IBM Power".

Reply:
Also if the region is verbosely "dal12" then maybe don't capitalize the title?